docs/index.md

```diff
@@ -60,5 +65,22 @@ The Origin Backend is a NestJS application that provides services to manage auth
 
 See documentation for the Origin Backend application [here](./backend.md). 
 
+## **Origin UI**
+[Source code on GitHub](https://github.com/energywebfoundation/origin/tree/master/packages/ui)
+
+The Origin UI package provides the code for the full, end-to-end I-REC-compliant reference implementation, as well as modularized libraries for each interface component. **Note that the UI uses the I-REC implementation of the SDKs,** so these serve more as a reference guide for generic implementations rather than for direct use. See an overview of the reference implementation [here](./user-guides.md)in the User Guides section. 
+
+All UI components are built with [React](https://reactjs.org/). For data fetching, UI components use [React Query](https://react-query.tanstack.com/overview) to access each SDK's controller methods. 
+
+### UI Libraries
+UI libraries provide that is modularized according to each reference implementation view:  
+
++ [certificate](https://github.com/energywebfoundation/origin/tree/master/packages/ui/libs/certificate)
++ [device](https://github.com/energywebfoundation/origin/tree/master/packages/ui/libs/device)
++ [exchange](https://github.com/energywebfoundation/origin/tree/master/packages/ui/libs/exchange)
++ [organization](https://github.com/energywebfoundation/origin/tree/master/packages/ui/libs/organization)
++ [user](https://github.com/energywebfoundation/origin/tree/master/packages/ui/libs/user)
++ [main UI](https://github.com/energywebfoundation/origin/tree/master/packages/ui/libs/ui)
```

Comment:
root ui is :
https://github.com/energywebfoundation/origin/tree/master/packages/ui/apps/origin-ui

Reply:
Still not fixed